engine/table/src/main/java/io/deephaven/engine/util/PythonScopeJpyImpl.java

```diff
@@ -195,12 +244,25 @@ private static CallableWrapper parseNumbaVectorized(PyObject pyObject, String nu
      * @return a Java object representing the underlying JPy object.
      */
     public static Object convert(PyObject pyObject) {
+        HashMap convertedMap = currentConvertedMap();
+        Object converted = convertedMap.get(pyObject);
```

Comment:
Does `PyObject` properly support the requirements for a `HashMap` key?

Reply:
Looking at the jpy docs, it looks like this works with pointer equality and hashing.  Is this what you want?
https://jpy.readthedocs.io/en/0.7.2/_static/java-apidocs/org/jpy/PyObject.html

Reply:
yes, exactly what I want and need.